index.html

```diff
@@ -75,5 +75,251 @@ <h4>LETS CONNECT</h4>
+                    <div>
+                      <h3>Multi-Post <br/> Stories</h3>
+                    </div>
+                    <div class="canopy-para">
+                      <p><span>CANOPY</span> <img src="./assets/Counter.png" alt="a picture dot"> Back end dev <img src="./assets/Counter.png" alt="a picture dot"> 2015</p>
+                    </div>
+                    <div class="mini-description">
+                      <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                      </p>
+                    </div>
+                    <div class="language-tags">
+                      <ul>
+                        <div>
+                          <li>html</li>
+                        </div>
+                        <div>
+                          <li>css</li>
+                        </div>
+                        <div>
+                          <li>javaScript</li>
+                        </div>
+                      </ul>
+                    </div>
+                    <div class="project-detail-btn">
+                      <button>See Project</button>
+                    </div>
+                  </div>
+                </div>
+             </div>
+       </section>
+      <section class="about-myself-section">
+        <div class="about-myself-description">
+          <div>
+            <h2>About <br/> Myself</h2>
+          </div>
+          <div>
+            <p>
+               Hello i’m a software developer! I can help you build a product , feature or website Look through some of my work and
+              experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+          </div>
+          <div class="lets-connect">
+            <h4>LETS CONNECT</h4>
+            <div class="social-icons about-me-icons">
+              <ul>
+                <li>
+                  <a href="#">
+                    <i class="fa-brands fa-github"></i>
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <i class="fa-brands fa-linkedin-in"></i>
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <i class="fa-regular fa-hand-peace"></i>
+                    
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <i class="fa-brands fa-twitter"></i>
+                  </a>
+                </li>
+                <li>
+                  <a href='#'>
+                    <i class="fa-solid fa-m"></i>
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div class="project-detail-btn">
+            <button>Get my resume</button>
+          </div>
+        </div>  
+        <div class="about-myself-skills">
```

Comment:
- [ ] Good job for implementing the accordions part, but please make sure to remove the extra space from the right.


![image](https://user-images.githubusercontent.com/34779928/192034197-87b415a4-351a-4c4c-9b54-9efee5ecdaa1.png)

Reply:
hey @aimalamiri, ive made the changes but i'm getting an unexpected lighthouse linter error, I don't know why.
All i did was update the css file. Please assist me with this.
Error: Process completed with exit code 1.